packages/element-library/src/media/editPanMoveable.js

```diff
@@ -61,6 +61,8 @@ function EditPanMoveable({
     moveableRef.current?.updateRect();
   });
 
+  useEffect(update, [update]);
```

Comment:
This runs on load, but also if the flip toggles changes while the edit-mode is mounted, which fixes the second bug mentioned.